### PR TITLE
Feature/task4 testing lecture

### PR DESCRIPTION
**Pull request by KorenYa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
   "scripts": {
     "start": "webpack-dev-server --open --mode development",
     "build": "webpack --mode production",
-    "test": "jest /src"
+    "test": "jest /src",
+    "e2e": "cypress open"
```
Comment (anniege18) [inline package.json]: Please add also script for coverage e.g. test:coverage

File: src/components/ErrorBoundary/index.test.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import renderer from 'react-test-renderer';
```
Comment (anniege18) [inline src/components/ErrorBoundary/index.test.js:2]: you could simply choose one of the tool - react-test-renderer or enzyme for all tests. it 's not necessary to use both

**Review by anniege18 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (KorenYa) [reply package.json]: Done.

File: src/components/ErrorBoundary/index.test.js (hunk 2 above)
Comment (KorenYa) [reply src/components/ErrorBoundary/index.test.js:2]: Will leave for example.

**Pull request merged**